app/src/main/java/io/bloco/circles/ui/auth/signup/ChooseUsernameActivity.kt

```diff
@@ -93,11 +98,16 @@ class ChooseUsernameActivity : BaseActivity() {
                     ChooseUsernameViewModel.Error.SignUpError -> getString(R.string.error)
                     ChooseUsernameViewModel.Error.InvalidUsername -> getString(R.string.invalid_username)
                 }
-
             }
             .launchIn(lifecycleScope)
     }
 
+    private fun showSkipSnackbar() = Snackbar
+        .make(root, R.string.continue_no_username_info, Snackbar.LENGTH_LONG)
+        .setAction(android.R.string.ok) {
+            GlobalScope.launch { viewModel.skipUsername() }
```

Comment:
Why is `skipUsername` a suspend method? Shouldn't we just be calling a regular `viewModel.usernameSkipped()` method?